### ISSUE TITLE
Example?

### ISSUE THREAD

**Issue by rborn** [yutin1987/react-native-bridge-firebase#1]
Would be amazing if you could create an example (now it's the default app 😄 ).
Thank you!

**Comment by xzilja** [comment]
@rborn by having a look at src of the project it should be along the lines of

```
import { Analytics } from 'react-native-bridge-firebase';

Analytics.logEvent('Some-event-name')
```

Similarly to ios / android api in firebase docs, for other features like auth and database you would need to use default firebase sdk for web. @yutin1987 am I right here, if so I can spend few hours this weekend writing up documentation :) I think you are really onto something here, will give this library a shot, hopefully it is as good as it looks like 👍

**Comment by yutin1987** [comment]
yes, you are right! only support Analytics, Config, Crash, Message (FCM), other features can use default firebase sdk for web.

@IljaDaderko, Thank you for your documentation.

maybe, I need more example info in https://github.com/yutin1987/react-native-bridge-firebase/tree/master/example. :)

**Comment by xzilja** [comment]
@yutin1987 yep, I think https://github.com/yutin1987/react-native-bridge-firebase/tree/master/example is just default react-native app at the moment? If you could add implementation examples there I can update README or another documentation file of your choice with some explanation and guides 👍
